tests/integration.sh

```diff
@@ -1322,7 +1341,13 @@ function test_send_from() {
     log 'sending funds from "a", using "b", to the Receiver, with message to the Receiver'
     local receiver_msg='{"increment":{}}'
     receiver_msg="$(base64 <<<"$receiver_msg")"
-    local send_message='{"send_from":{"owner":"'"${ADDRESS[a]}"'","recipient":"'"$receiver_addr"'","amount":"400000","msg":"'$receiver_msg'"}}'
+
+    if [ "$skip_register_receiver" = "true" ]; then
+        send_message='{"send_from":{"owner":"'"${ADDRESS[a]}"'","recipient":"'"$receiver_addr"'","recipient_code_hash":"'"$receiver_hash"'","amount":"400000","msg":"'$receiver_msg'"}}'
+    else
+        send_message='{"send_from":{"owner":"'"${ADDRESS[a]}"'","recipient":"'"$receiver_addr"'","amount":"400000","msg":"'$receiver_msg'"}}'
+    fi
+
```

Comment:
same as above, please add a declaration of the local variable.

Reply:
Done